src/DynamoCore/DynamoCore.csproj

```diff
@@ -21,10 +21,10 @@
     <Reference Include="System.Configuration" />
   </ItemGroup>
   <ItemGroup>
-	<PackageReference Include="DynamoVisualProgramming.LibG_227_0_0" Version="2.13.0.6926" GeneratePathProperty="true">
+	<PackageReference Include="DynamoVisualProgramming.LibG_227_0_0" Version="2.13.0.7687" GeneratePathProperty="true">
```

Comment:
Not related to this PR but I think it's time to bump up the release version to 2.14 in LibG.